packages/dd-trace/test/config.spec.js

```diff
@@ -15,6 +16,13 @@ describe('Config', () => {
   let existsSyncReturn
   let osType
 
+  const RULES_JSON_PATH = require.resolve('./fixtures/config/appsec-rules.json')
+  const RULES_JSON = require(RULES_JSON_PATH)
+  const BLOCKED_TEMPLATE_HTML_PATH = require.resolve('./fixtures/config/appsec-blocked-template.html')
+  const BLOCKED_TEMPLATE_JSON_PATH = require.resolve('./fixtures/config/appsec-blocked-template.json')
+  const BLOCKED_TEMPLATE_HTML = '<html>blocked</html>' + EOL
+  const BLOCKED_TEMPLATE_JSON = '{"error": "blocked"}' + EOL
```

Comment:
at this point, just use require like for RULES_JSON_PATH
```suggestion
  const BLOCKED_TEMPLATE_HTML = require(BLOCKED_TEMPLATE_HTML_PATH)
  const BLOCKED_TEMPLATE_JSON = require(BLOCKED_TEMPLATE_JSON_PATH)
```

Reply:
I think you cannot require an .html and the result of require a json is an object not a string, so you need to stringify it

Reply:
```js
 fs.readFileSync(BLOCKED_TEMPLATE_HTML_PATH).toString()
```

Reply:
whoops :facepalm: i didn't notice it's an actual html file

Reply:
```suggestion
  const BLOCKED_TEMPLATE_HTML = fs.readFileSync(BLOCKED_TEMPLATE_HTML_PATH, { encoding: 'utf8' })
  const BLOCKED_TEMPLATE_JSON = require(BLOCKED_TEMPLATE_JSON_PATH)
```